src/Info.js

```diff
@@ -0,0 +1,42 @@
+import { calculateWinner } from './Utils.js';
+
+export function Status({ currentStep, xIsNext }) {
```

Comment:
In terms of project structure, you should keep each component in its own file. I will recommend you move the `Status` component to a different file and import it into the `Moves` component.